vendor/modules.txt

```diff
@@ -13,23 +13,23 @@ github.com/blang/semver
 # github.com/certifi/gocertifi v0.0.0-20190905060710-a5e0173ced67
 github.com/certifi/gocertifi
 # github.com/coreos/etcd v3.3.15+incompatible
-github.com/coreos/etcd/auth/authpb
 github.com/coreos/etcd/clientv3
+github.com/coreos/etcd/auth/authpb
 github.com/coreos/etcd/clientv3/balancer
```

Comment:
Nit In general, I think it would be good for this file to only contain changes to modules you are adding. While this was done with automated tooling it is confusing to tell what actually changed here.

Reply:
dropping this change could very well be differences in tool versions